### PR TITLE
Merge utils3 to utils

### PR DESCRIPTION
**Pull request by oeway**
We will drop support of python2x, so we can have a single utils file.

### CODE REVIEW

File: python/imjoy_rpc/utils.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 """Provide utils for Python 2 plugins."""
```
Comment (MartinHjelmare) [inline python/imjoy_rpc/utils.py:1]: Stale docstring.

File: python/imjoy_rpc/__main__.py [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
 
 class ImJoyPlugin:
-    def __init__(self, api):
-        self._api = api
-
-    def setup(self):
-        self._api.alert("hello")
-        self._api.showDialog(type="external", src="https://imjoy.io")
+    async def setup(self):
+        await api.log('plugin initialized')
```
Comment (MartinHjelmare) [inline python/imjoy_rpc/__main__.py:6]: Are we running black?

Comment (oeway) [reply python/imjoy_rpc/__main__.py:6]: We should, maybe I forgot

**Comment by oeway** [comment]
BTW @MartinHjelmare I will keep the long rpc.py file for now to keep consistency with the JS implementation. Will see how to refactor later, hopefully have the same organisation in JS and Python.

**Pull request merged**